docs/how-to-guides/azure_resource_provision.json

```diff
@@ -39,6 +39,52 @@
       "metadata": {
         "description": "Whether or not to deploy eventhub provision script"
       }
+    },
+    "databaseServerName": {
+      "type": "string",
+      "defaultValue": "[concat('server-', uniqueString(resourceGroup().id, deployment().name))]",
```

Comment:
It's not showing up in the ARM template correctly (maybe I did something wrong on my end). can you attach a screenshot on the ARM template page?

Reply:
![image](https://user-images.githubusercontent.com/148079/174251470-acbb07ee-e39d-4644-b777-e5dbd008cace.png)
It shows correctly here